dhall/src/Dhall/Import.hs

```diff
@@ -746,6 +748,26 @@ toHeader _ = do
 getCacheFile
     :: (Alternative m, MonadIO m) => FilePath -> Dhall.Crypto.SHA256Digest -> m FilePath
 getCacheFile cacheName hash = do
+    cacheDirectory <- getOrCreateCacheDirectory cacheName
+
+    let cacheFile = cacheDirectory </> ("1220" <> show hash)
+
+    return cacheFile
+
+warnAboutMissingCache :: (Alternative m, MonadIO m) => m ()
+warnAboutMissingCache = alternative₀ <|> alternative₁
+  where
+    alternative₀ = getOrCreateCacheDirectory "dhall" >> return ()
```

Comment:
We should also warn if the `dhall-haskell` subdirectory does not exist, since it is used for the semi-semantic cache